crypto/tss/ecdsa/cggmp/refresh/refresh_test.go

```diff
@@ -35,7 +35,7 @@ func TestRefresh(t *testing.T) {
 
 var (
 	threshold = uint32(2)
-	publicKey = pt.ScalarBaseMult(elliptic.Secp256k1(), big1)
```

Comment:
add one var:
secret = big.NewInt(2)
publicKey = pt.ScalarBaseMult(elliptic.Secp256k1(), secret)

Reply:
change Line58:
shareA := big.NewInt(3)
change Line 63:
shareB := big.NewInt(4)
delete Line 71 : secret := big.NewInt(1).

According to your change :: private key is 2, 
the chosen polynomial becomes f(x) =  x+2 (i.e. f(0) = 2).

Therefore, the share of A is f(1) = 3 (ref. Line 83 Birkhoff coefficient is (x-coord, rank) = (1,0)).
the share of B is f(2) = 4. (ref. Line 84 Birkhoff coefficient is (x-coord, rank) = (2,0)).

The original case, the polynomial is g(x) = x+1, so the corresponding public key is G, 
share A = g(1) = 2 and share B = g(2)= 3.

Thanks a lot!